lib/core/context.c

```diff
@@ -190,3 +190,28 @@ void h2o_context_update_timestamp_string_cache(h2o_context_t *ctx)
     h2o_time2str_rfc1123(ctx->_timestamp_cache.value->rfc1123, &gmt);
     h2o_time2str_log(ctx->_timestamp_cache.value->log, ctx->_timestamp_cache.tv_at.tv_sec);
 }
+
+int h2o_context_close_idle_connections(h2o_context_t *ctx, int max_connections_to_close, int min_age)
```

Comment:
Regarding `min_age`, I think we might want to use `uint64_t` representing milliseconds. I think that's our common pattern in the public API of libh2o (see various timeouts in h2o.h).
Regarding the type of `max_connections_to_close`, please see my comment above.